src/jit/emitxarch.cpp

```diff
@@ -1306,6 +1286,11 @@ bool emitter::EncodedBySSE38orSSE3A(instruction ins)
 
     size_t insCode = 0;
 
+    if (!IsSSEOrAVXInstruction(ins))
```

Comment:
This covers the additional check that `Is4ByteSSE4Instruction` and `Is4ByteSSE4OrAVXInstruction` were doing, but as a single check (rather than multiple separate ones).
It also short-circuits the path for any instruction that can't be SSE38/SSE3A.